test-cloud/pom.xml

```diff
@@ -128,8 +131,33 @@
             </systemProperties>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-resources-plugin</artifactId>
+          <executions>
+            <execution>
+              <id>process-resources</id>
+              <phase>generate-resources</phase>
+              <goals>
+                <goal>resources</goal>
+              </goals>
+            </execution>
+          </executions>
+        </plugin>
       </plugins>
     </pluginManagement>
+    <resources>
+      <resource>
+        <directory>src/main/filtered-resources</directory>
+        <filtering>true</filtering>
+      </resource>
+    </resources>
+    <plugins>
```

Comment:
Isn't maven-resources-plugin part of the build by default (so you could skip this `<plugins>` part)?

Reply:
Not here because test-cloud module has pom packaging.